pr-data.csv

```diff
@@ -1396,125 +1396,125 @@ https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modu
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortWriteRemovesUncommittedVersion,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.abortWriteReturnsTheRemovedVersion,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.addWriteReturnsNullIfNoUncommittedVersionExists,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.addWriteReturnsUncommittedVersionIfItExists,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.addWriteWithDifferentTxIdThrows,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.afterRemovalPreviousVersionRemainsAccessibleByTimestamp,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.afterRemovalReadByLatestTimestampFindsNothing,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.afterRemovalReadWithTxIdFindsNothing,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
+https://github.com/apache/ignite-3,a1aa7fd4e2398c72827777ec5417d67915ff4da3,modules/storage-page-memory,org.apache.ignite.internal.storage.pagememory.mv.PageMemoryMvPartitionStorageTest.committedMultiPageValuesAreReadSuccessfully,ID,Deleted,,https://github.com/apache/ignite-3/commit/2a6d30702c873bd61b6230784d86a9adfcd4d4cd
```

Comment:
Why are the four tests (1443, 1444, 1468, 1469) `Deleted`, not `MovedOrRenamed`?

Reply:
Agree, put `Deleted` here might be better